cookbook/import_export/automate-import-exports.rst

```diff
@@ -8,14 +8,14 @@ The batch job command
 
 Akeneo PIM provides a simple command to launch jobs:
 
-.. code-block:: batch
+.. code-block:: bash
     :linenos:
 
     app/console akeneo:batch:job [-c|--config CONFIG] [--email EMAIL] [--no-log] [--] <code>
```

Comment:
shouldn't we do what we say then ? imho I would  'not forget' to write the --env=prod. But it's a detail.